CMakeLists.txt

```diff
@@ -174,4 +174,6 @@ add_subdirectory(source)
 add_subdirectory(tools)
 add_subdirectory(test)
 
-FetchContent_MakeAvailable(freertos_kernel cmock)
+if(PROJECT_IS_TOP_LEVEL)
```

Comment:
`PROJECT_IS_TOP_LEVEL` is new for CMake 3.21. I think we'll need to bump the minimum required version of CMake at the top of this file.

Reply:
done.

Reply:
Thank you!